src/selectors/prescriber.js

```diff
@@ -45,5 +45,13 @@ export const selectSortedPrescribers = createSelector(
 
 export const selectPrescriberModalOpen = ({ prescriber }) => {
   const { isCreatingPrescriber, isEditingPrescriber } = prescriber;
+
   return isCreatingPrescriber || isEditingPrescriber;
 };
+
+export const selectCanEditPrescriber = ({ prescriber }) => {
+  const { currentPrescriber } = prescriber;
+  const { fromThisStore } = currentPrescriber ?? {};
+
+  return fromThisStore ?? true;
```

Comment:
Shouldn't this default to `false`?

Reply:
default to true for when the modal is opened and there isn't actually a prescriber (creation)